tests/e2e/specs/schemas.spec.ts

```diff
@@ -33,102 +24,49 @@ import { openConfluentSidebar } from "../utils/sidebarNavigation";
  */
 
 test.describe("Schema Management", () => {
-  let resourcesView: ResourcesView;
-  // this is set after the connections are set up based on their beforeEach hooks
-  let schemasView: SchemasView;
-  let notificationArea: NotificationArea;
```

Comment:
Not necessary here since we aren't dealing with singletons and it's easy enough to set up new page object instances